python/Ganga/Utility/execute.py

```diff
@@ -2,16 +2,28 @@
 import base64
 import subprocess
 import threading
-import pickle
+import cPickle as pickle
 import signal
+import shutil
+import tempfile
 from Ganga.Core.exceptions import GangaException
 from Ganga.Utility.logging import getLogger
+from exceptions import RuntimeError
```

Comment:
There's no need to import anything. All the python standard exceptions are builtins